content/en/real_user_monitoring/frustration_signals/_index.md

```diff
@@ -0,0 +1,153 @@
+---
+title: Frustration Signals
+kind: documentation
+further_reading:
+- link: '/real_user_monitoring/dashboards/frustration_signals_dashboard/'
+  tag: 'Documentation'
+  text: 'Frustration Signals Dashboard'
+- link: '/real_user_monitoring/explorer'
+  tag: 'Documentation'
+  text: 'Learn about the RUM Explorer'
+- link: '/real_user_monitoring/session_replay'
+  tag: 'Documentation'
+  text: 'Learn about Session Replay'
+---
+
+## Overview
+
+<div class="alert alert-warning">
+For access to frustration signals, contact <a href="/help">Datadog Support</a>.
+</div>
+
+Frustration signals identify the highest points of user friction in your application by surfacing moments where users exhibit frustration. 
+
+RUM collects three types of frustration signals:
+
+Rage Clicks
+: A user clicks on an element more than three times in a one-second rolling window.
+
+Dead Clicks
+: A user clicks on a static element thinking it is interactive.
+
+Error Clicks
+: A user clicks on an element right before a JavaScript error occurs.
+
+## Requirements
+
+First, you need the Browser RUM SDK version >= 4.9.0.
+
+To start collecting frustration signals, add the following to your SDK configuration:
+
+```
+DD_RUM.init({
+  trackInteractions: true,
+  trackFrustrations: true
+})
+```
+
+Frustration signals require actions. Enabling `trackFrustrations` automatically enables `trackInteractions`.
+
+## Usage
+
+Frustration signals appear as high-level data points representing sources of user frustration on the [**RUM Applications** page][1]. The [RUM Explorer][2] displays a count of frustration signals as a column in the **List** visualization. 
```

Comment:
not sure where the second sentence came from, we can remove. it's not true unless a user does this themselves

Reply:
```suggestion
Frustration signals appear as high-level data points representing sources of user frustration on the [**RUM Applications** page][1]. To display a list of frustration signals count in the [RUM Explorer][2], click the **Options** button and add a column with `@session.frustration.count`. 
```